locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "Copy": "नक़ल",
+  "Erase": "मिटाएँ",
+  "Play": "बजाएं",
+  "Stop": "रुकें",
+  "Paste": "चिपकाएँ",
+  "Save block help": "ब्लॉक सहायता सहेजें",
+  "ti la sol fa mi re do": "ती ला सोल फा मी रे दो",
+  "Jumping to the bottom of the page.": "पृष्ठ के नीचे जाना।",
+  "Scrolling up.": "ऊपर स्क्रोल कर रहा है।",
+  "Scrolling down.": "नीचे स्क्रोल कर रहा है।",
+  "Extracting block": "ब्लॉक निकाल रहा है",
+  "Moving block up.": "ब्लॉक को ऊपर मोव कर रहा है।",
+  "Moving block down.": "ब्लॉक को नीचे मोव कर रहा है।",
+  "Moving block left.": "ब्लॉक को बाएँ मोव कर रहा है।",
+  "Moving block right.": "ब्लॉक को दाएँ मोव कर रहा है।",
+  "Jump to home position.": "होम स्थिति पर जाएँ।",
+  "Hide blocks": "ब्लॉक छुपाएँ",
+  "Trash can is empty.": "कचरा पात्र खाली है।",
+  "Item restored from the trash.": "आइटम को कचरे से पुनः स्थापित किया गया।",
+  "Restore last item": "आखिरी आइटम पुनः स्थापित करें",
+  "Restore all items": "सभी आइटम पुनः स्थापित करें",
+  "Click the run button to run the project.": "परियोजना चलाने के लिए रन बटन पर क्लिक करें।",
+  "Home": "होम",
+  "Show/hide blocks": "ब्लॉक्स दिखाएँ/छुपाएँ",
+  "Expand/collapse blocks": "ब्लॉक विस्तार करें/संकुचित करें",
+  "Decrease block size": "ब्लॊक का आकार घटाएं",
+  "Increase block size": "ब्लॊक का आकार बढ़ाएं",
+  "Could not parse JSON input.": "JSON इनपुट को पार्स नहीं कर सका।",
+  "Select is enabled.": "चयन सक्षम है।",
+  "Select is disabled.": "चयन अक्षम है।",
+  "Cannot load project from the file. Please check the file type.": "फ़ाइल से परियोजना लोड नहीं कर सकता है। कृपया फ़ाइल का प्रकार जांचें।",
+  "Invalid parameters": "अमान्य पैरामीटर्स",
+  "Error regenerating palettes. Please refresh the page.": "पैलेट्स को फिर से बनाने में त्रुटि हुई। कृपया पृष्ठ रीफ्रेश करें।",
+  "temperament": "स्वभाव",
+  "start": "प्रारंभ",
+  "matrix": "मैट्रिक्स",
+  "status": "स्थिति",
+  "drum mapper": "ड्रम मैपर",
+  "ruler": "रूलर",
+  "timbre": "टिम्बर",
+  "stair": "सीढ़ी",
+  "tempo": "टेम्पो",
+  "mode": "मोड",
+  "slider": "स्लाइडर",
+  "keyboard": "कुंजीपट",
+  "drum": "ढोल",
+  "rhythm maker": "रिदम बनाने वाला",
+  "note value": "नोट मूल्य",
+  "scalar interval": "स्केलर इंटरवल",
+  "milliseconds": "मिलीसेकंड",
+  "silence": "मौन",
+  "down": "नीचे",
+  "up": "ऊपर",
+  "pitch": "पिच",
+  "Silence block cannot be removed.": "मौन अवरोध हटाया नहीं जा सकता",
+  "You can restore deleted blocks from the trash with the Restore From Trash button.": "आप हटाए गए ब्लॉकों को कचरे से पुनर्स्थापित कर सकते हैं 'कचरे से पुनर्स्थापित करें' बटन द्वारा।",
+  "true": "सत्य",
+  "false": "असत्य",
+  "Cartesian": "कार्टेसियन",
+  "polar": "पोलर",
+  "Cartesian/Polar": "कार्टेसिय",
+  "none": "कोई नहीं",
+  "treble": "ट्रेबल",
+  "grand staff": "ग्रैंड स्टाफ",
+  "mezzo-soprano": "मेज़्ज़ो-सोप्रानो",
+  "alto": "आल्टो",
+  "tenor": "टेनर",
+  "bass": "बास",
+  "on2": "ऑन2",
+  "off": "ऑफ",
+  "Not a number": "एक संख्या नहीं",
+  "Octave value must be between 1 and 8.": "ऑक्टेव मान 1 से 8 के बीच होना चाहिए।",

```

perhaps "ऑक्टेव का मान" will be better?
```suggestion
  "Octave value must be between 1 and 8.": "ऑक्टेव का मान 1 से 8 के बीच होना चाहिए।",

```